service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/ServiceLocator.java

```diff
@@ -32,14 +32,40 @@
     }
 
     /**
-     * Locate a service with the given name.
+     * Locate the service's URIs for the given name.
+     *
+     * @param name The name of the service.
+     * @return One or more URIs for that service, otherwise an empty List if none is found.
+     */
```

Comment:
Maybe we should put some `@since 1.4` tags in. We haven't been very consistent about this, but I think it's a nice thing to do.